pkg/cmd/clients.go

```diff
@@ -172,6 +172,9 @@ When used standalone, a namespace must be specified by providing the --namespace
 			case "cordova":
 				app.Annotations["icon"] = "font-icon icon-cordova"
 				break
+			case "xamarin":
+				app.Annotations["icon"] = "font-icon icon-xamarin"
+				break
```

Comment:
I know this was not part of the work, but I think adding a default clause here would make sense and error out with "unknown client type"